SDK-v4/gdpr/app/src/main/res/values/localytics.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why do we have a Localytics.xml for SDK-v4. It may be best to show the change to an existing sample so people can tell what exactly they need to do.

Reply:
Getting deleted now.